README.md

```diff
@@ -14,6 +14,31 @@ Run `npm test`
 to the following object (`{memory: true}`) being passed in as a parameter for new 
 sqliteDatabase() located in `searchDao.js`, AND all `Spec js` files in the test directory.
 
+## Documentation
+
+When making a GET call, you can add query parameters. These are all of the parameters you 
+can add.
+```
```

Comment:
In my experience, not having a blank line before the beginning or after the end of a triple-backtick-quoted block leads to results that are inconsistent at best; I'm surprised that this seems to be formatted properly as it stands, but for more reliably correct rendering I recommend adding blank lines before and after this block.

Reply:
Okay, will do.